wrangler.toml

```diff
@@ -1,30 +1,30 @@
-name = "dns-over-discord"
-type = "javascript"
-account_id = "fb1f542488f2441acf88ca15f3a8390d"
-workers_dev = true
-compatibility_date = "2021-10-17"
-kv_namespaces = [
-    { binding = "CACHE", id = "49442b14c2cb482ea086750526022d3e", preview_id = "49442b14c2cb482ea086750526022d3e" }
-]
-
-[build]
-command = "npm run build"
-
-[build.upload]
-format = "service-worker"
-
-[env.staging]
-zone_id = "63feb9b9ee057b117f7c833928b0d21f"
-workers_dev = false
-route = "dns-over-discord-staging.v4.wtf/*"
-kv_namespaces = [
-    { binding = "CACHE", id = "a4e14492f7624d39810022793969428b" }
-]
-
-[env.production]
-zone_id = "63feb9b9ee057b117f7c833928b0d21f"
-workers_dev = false
-route = "dns-over-discord.v4.wtf/*"
-kv_namespaces = [
-    { binding = "CACHE", id = "ac1c24f2c9594a97aeb7dd30377b45ef" }
-]
+name = "dns-over-discord"
+type = "javascript"
+account_id = "fb1f542488f2441acf88ca15f3a8390d"
+workers_dev = true
+compatibility_date = "2021-10-17"
+kv_namespaces = [
+    { binding = "CACHE", id = "49442b14c2cb482ea086750526022d3e", preview_id = "49442b14c2cb482ea086750526022d3e" }
+]
+
+[build]
+command = "npm run build"
+
+[build.upload]
+format = "service-worker"
+
+[env.staging]
+zone_id = "63feb9b9ee057b117f7c833928b0d21f"
+workers_dev = false
+route = "dns-over-discord-staging.v4.wtf/*"
+kv_namespaces = [
+    { binding = "CACHE", id = "a4e14492f7624d39810022793969428b" }
+]
+
+[env.production]
+zone_id = "63feb9b9ee057b117f7c833928b0d21f"
+workers_dev = false
+route = "dns-over-discord.v4.wtf/*"
+kv_namespaces = [
+    { binding = "CACHE", id = "ac1c24f2c9594a97aeb7dd30377b45ef" }
+]
```

Comment:
This file still has an unexpected diff

Reply:
Why? Should just be a case of resetting the file to as it is in master? `git checkout master -- wrangler.toml`